Console Errors

- [x]  https://github.com/sponsors/sindresorhus

```js
❌ forked-to
content.js:2856 TypeError: Cannot read property 'addEventListener' of null
    at forked_to_init (content.js:7820)
    at features_run (content.js:2851)
    at content.js:2898
    at content.js:2813
    at onAjaxedPagesRaw (content.js:2808)
```
```js
❌ more-dropdown
content.js:2856 TypeError: Cannot read property 'append' of null
    at appendBefore (content.js:3537)
    at createDropdown (content.js:3565)
    at more_dropdown_init (content.js:3578)
    at async features_run (content.js:2851)
```
```js
❌ edit-files-faster
content.js:2856 Error: Unable to fetch.
Ensure that your token has access to this repo.
{
	"message": "Not Found",
	"documentation_url": "https://developer.github.com/v3/repos/#get"
}
    at getError (content.js:3063)
    at async chrome-extension:/hlepfoohegkhhmjieoechaddaejaokhf/content.js:3025
    at async fetchFromApi (content.js:3078)
    at async get_default_branch (content.js:3086)
    at async edit_files_faster_init (content.js:6973)
    at async features_run (content.js:2851)
```
- [x]  Click on the sponsor button in a repo or the watch dropdown
```js
content.js:4502 Uncaught TypeError: Failed to execute 'observe' on 'IntersectionObserver': parameter 1 is not of type 'Element'.
    at HTMLDocument.<anonymous> (content.js:4502)
    at HTMLDocument.listenerFn (content.js:369)
```

- [ ] https://gist.github.com/yakov116
```js
❌ embed-gist-via-iframe
content.js:2856 TypeError: Cannot read property 'cloneNode' of null
    at embed_gist_via_iframe_init (content.js:8398)
    at features_run (content.js:2851)
    at Object.add (content.js:2896)
```

--- a/source/features/embed-gist-via-iframe.tsx
+++ b/source/features/embed-gist-via-iframe.tsx
@@ -27,7 +27,7 @@ features.add({
 	description: 'Adds a menu item to embed a gist via <iframe>.',
 	screenshot: 'https://user-images.githubusercontent.com/44045911/63633382-6a1b6200-c67a-11e9-9038-aedd62e4f6a8.png',
 	include: [
-		features.isGist
+		features.isSingleGist
 	],
 	load: features.onDomReady,
 	init

--- a/source/features/embed-gist-via-iframe.tsx
+++ b/source/features/embed-gist-via-iframe.tsx
@@ -1,6 +1,8 @@
 import select from 'select-dom';
 import features from '../libs/features';
 
+const isSingleGist = (): boolean => features.isGist() && /^\/(gist\/)?[^/]+\/[0-9a-f]{32}$/.test(location.pathname);
+
 function init(): void {
 	const embedViaScript = select('.file-navigation-option [value^="<script"]')!;
 	const embedViaIframe = embedViaScript.cloneNode(true) as HTMLButtonElement;
@@ -27,7 +29,7 @@ features.add({
 	description: 'Adds a menu item to embed a gist via <iframe>.',
 	screenshot: 'https://user-images.githubusercontent.com/44045911/63633382-6a1b6200-c67a-11e9-9038-aedd62e4f6a8.png',
 	include: [
-		features.isSingleGist
+		isSingleGist
 	],
 	load: features.onDomReady,
 	init

--- a/source/features/embed-gist-via-iframe.tsx
+++ b/source/features/embed-gist-via-iframe.tsx
@@ -1,8 +1,6 @@
 import select from 'select-dom';
 import features from '../libs/features';
 
-const isSingleGist = (): boolean => features.isGist() && /^\/(gist\/)?[^/]+\/[0-9a-f]{32}$/.test(location.pathname);
-
 function init(): void {
 	const embedViaScript = select('.file-navigation-option [value^="<script"]')!;
 	const embedViaIframe = embedViaScript.cloneNode(true) as HTMLButtonElement;
@@ -29,7 +27,7 @@ features.add({
 	description: 'Adds a menu item to embed a gist via <iframe>.',
 	screenshot: 'https://user-images.githubusercontent.com/44045911/63633382-6a1b6200-c67a-11e9-9038-aedd62e4f6a8.png',
 	include: [
-		isSingleGist
+		features.isSingleGist
 	],
 	load: features.onDomReady,
 	init

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -325,6 +325,10 @@ export const _isSingleFile = [
 	'https://github.com/sindresorhus/refined-github/blob/fix-narrow-diff/distribution/content.css',
 	'https://github.com/sindresorhus/refined-github/blob/master/edit.txt'
 ];
+export const isSingleGist = (): boolean => isGist() && /^\/(gist\/)?[^/]+\/[0-9a-f]{32}$/.test(location.pathname);
+export const _isSingleGist = [
+	'https://gist.github.com/sindresorhus/0ea3c2845718a0a0f0beb579ff14f064'
+];
 
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');
 export const _isTrending = [

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -326,11 +326,6 @@ export const _isSingleFile = [
 	'https://github.com/sindresorhus/refined-github/blob/master/edit.txt'
 ];
 
-export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option');
-export const _isSingleGist = [
-	// TODO: add test URLs
-];
-
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');
 export const _isTrending = [
 	'https://github.com/trending',

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -326,6 +326,11 @@ export const isSingleFileTest = [
 	'https://github.com/sindresorhus/refined-github/blob/master/edit.txt'
 ];
 
+export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option')
+export const isSingleGistTest = [
+	'https://gist.github.com/yakov116/2d81580f7907839f9065e670f0aba64c'
+]
+
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');
 export const isTrendingTest = [
 	'https://github.com/trending',

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -326,10 +326,10 @@ export const isSingleFileTest = [
 	'https://github.com/sindresorhus/refined-github/blob/master/edit.txt'
 ];
 
-export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option')
+export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option');
 export const isSingleGistTest = [
 	'https://gist.github.com/yakov116/2d81580f7907839f9065e670f0aba64c'
-]
+];
 
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');
 export const isTrendingTest = [

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -327,9 +327,7 @@ export const isSingleFileTest = [
 ];
 
 export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option');
-export const isSingleGistTest = [
-	'https://gist.github.com/sindresorhus/0ea3c2845718a0a0f0beb579ff14f064'
-];
+export const isSingleGistTest = domBased;
 
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');
 export const isTrendingTest = [

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -328,7 +328,7 @@ export const isSingleFileTest = [
 
 export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option');
 export const isSingleGistTest = [
-	'https://gist.github.com/yakov116/2d81580f7907839f9065e670f0aba64c'
+	'https://gist.github.com/sindresorhus/0ea3c2845718a0a0f0beb579ff14f064'
 ];
 
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');

--- a/source/libs/page-detect.ts
+++ b/source/libs/page-detect.ts
@@ -327,7 +327,9 @@ export const _isSingleFile = [
 ];
 
 export const isSingleGist = (): boolean => isGist() && select.exists('.file-navigation-option');
-export const isSingleGistTest = domBased;
+export const _isSingleGist = [
+	// TODO: add test URLs
+];
 
 export const isTrending = (): boolean => location.pathname === '/trending' || location.pathname.startsWith('/trending/');
 export const _isTrending = [

